order/buyer/receive/single-delivery-order-response.md

```diff
@@ -2,251 +2,233 @@
 description: How to receive a single delivery order response sent by the supplier
 ---
 
-# Receive a single delivery order response
+# Receive a Single Delivery Order Response
```

Comment:
Same as the previous page:
It's not updated in Gitbooks. Most of the changes are not related to `originalPosition`.

Reply:
reverted below ~ `Order header`